controllers/auth/authcontroller.js

```diff
@@ -17,53 +18,62 @@ module.exports = {
             where: { email }
         }).then(user => {
 
-             if (!user){
-                res.status(404).json({ success: false, message: "Usuario con este correo no encontrado" });
-             } else {
+            if (!user){
+                responses.errorDTOResponse(res,404,"Usuario con este correo no encontrado");
+            } else {
```

Comment:
```suggestion
            if (!user)
                return responses.errorDTOResponse(res,404,"Usuario con este correo no encontrado");
```
Con esto, nos deshacemos de varios niveles de identación y sí usamos bien los guard clausures xd
Eso sí, revisar más adelante y corregir los niveles de identación, además de colocar los try - catch que estén implementados como callbacks